commit 357327cbd6c9c273446cb3f0c8edd49b9950be3a
Author: John Johansen <john.johansen@canonical.com>
Date:   Fri Mar 18 08:42:41 2016 -0700

    apparmor: add missing parens. not a bug fix but highly recommended
    
    Signed-off-by: John Johansen <john.johansen@canonical.com>
    Signed-off-by: Leann Ogasawara <leann.ogasawara@canonical.com>

diff --git a/security/apparmor/domain.c b/security/apparmor/domain.c
index a701534..ee4b6ef 100644
--- a/security/apparmor/domain.c
+++ b/security/apparmor/domain.c
@@ -731,7 +731,7 @@ int apparmor_bprm_set_creds(struct linux_binprm *bprm)
 	 * subsets are allowed even when no_new_privs is set because this
 	 * aways results in a further reduction of permissions.
 	 */
-	if (bprm->unsafe & LSM_UNSAFE_NO_NEW_PRIVS &&
+	if ((bprm->unsafe & LSM_UNSAFE_NO_NEW_PRIVS) &&
 	    !unconfined(label) && !aa_label_is_subset(new, label)) {
 		error = -EPERM;
 		info = "no new privs";
